Assignment2/src/Assignment2/Example1.java

```diff
@@ -0,0 +1,46 @@
+package Assignment2;
+import java.util.LinkedList;
+import java.util.List;
+public class Example1 
+{
+public static String destCity(List<List<String>> paths)
+{
+	List<String> outgoingElements=new LinkedList<>();
+	List<String> incomingElements=new LinkedList<>();
+	for(int i=0; i<paths.size();i++)
+	{
```

Comment:
Try to use For each loop if possible as it is more faster than normal for loop.

Reply:
Added For Each loop.